src/components/ApplicationsCards/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from 'react-rainbow-components/components/Card';
+import GithubIcon from '../icons/Github';
+import './styles.css';
+
+export default function ApplicationsCards({ results }) {
```

Comment:
I think should be useful to name this components as PrismicApplicationCards, always with the Prismic suffix to easily identify them. Don't change the class names.